sw/device/lib/testing/test_framework/ottf_flow_control.h

```diff
@@ -0,0 +1,57 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SW_DEVICE_LIB_TESTING_TEST_FRAMEWORK_OTTF_FLOW_CONTROL_H_
+#define OPENTITAN_SW_DEVICE_LIB_TESTING_TEST_FRAMEWORK_OTTF_FLOW_CONTROL_H_
+#include <stdint.h>
+
+#include "sw/device/lib/base/status.h"
+#include "sw/device/lib/dif/dif_uart.h"
+
+/**
+ * Flow control state.
+ */
+typedef enum flow_control {
+  /** No flow control enabled. */
+  kFlowControlNone = 0,
+  /** Automatically determine the next flow control state. */
+  kFlowControlAuto = 1,
```

Comment:
Nit: In case this is to be used as ascii, and this is mostly carrying text and not headers, should it be STX instead of SOH?

Reply:
This enum only encodes the current flow-control state and requested next state.  By coincidence, Pause and Resume happen to have the same values as ASCII XON and XOFF.

[XON and XOFF are the only ASCII codes](https://en.wikipedia.org/wiki/Software_flow_control) involved in actually pausing and resuming the data flow.